### PR TITLE
MET-5686 1000 Record Limit - allow cancellation of single poller

### PR DESCRIPTION
**Pull request by andyjmaclean**

### CODE REVIEW

**Review by jeortizquan — COMMENTED** [review]
The warning is displayed
afbeelding

The flag is also displayed
afbeelding

Have a look on the comments.

File: projects/sandbox/src/app/_mocked/mocked-sandbox.service.ts [hunk 1]
```diff
@@ -2,7 +2,7 @@ import { FormGroup } from '@angular/forms';
 import { Observable, of, throwError, timer } from 'rxjs';
 import { delay, switchMap } from 'rxjs/operators';
 import {
-  Dataset,
+  DatasetProgress,
```
[the remark this answers is not in the document]
Comment (andyjmaclean) [reply projects/sandbox/src/app/_mocked/mocked-sandbox.service.ts]: Fixed!

File: projects/shared/src/lib/data-polling/data-polling.component.spec.ts [hunk 2]
```diff
@@ -30,15 +30,18 @@ describe('DataPollingComponent', () => {
   });
 
   // Intantiate poller (needs called from the async context)
-  const initDefaultDataPoller = (errorMode?: boolean): PollingSubjectAccesor => {
+  const initDefaultDataPoller = (
+    errorMode?: boolean,
+    identifier?: string
+  ): PollingSubjectAccesor => {
```
[the remark this answers is not in the document]
Comment (andyjmaclean) [reply projects/shared/src/lib/data-polling/data-polling.component.spec.ts]: Well spotted.  Fixed!

**Review by jeortizquan — APPROVED** [review]
It displays the warning.
afbeelding

**Pull request merged**